src/reducers/index.js

```diff
@@ -55,8 +55,8 @@ import {
   FIND_OR_CREATE_THREAD,
   FINISH_LOGIN,
   HIDE_POPOVER,
+  LOCATION_CHANGE,
```

Comment:
Since we don't "own" this, just do the import from react-router-redux here and don't include it in our action constants above:
`import { LOCATION_CHANGE } from 'react-router-redux'`